Build env_logger without unnecessary features to reduce the binary size by
a factor of two.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,1 +18,1 @@
-env_logger = "0.9.0"
+env_logger = { version = "0.9.0", default-features = false, features = ["humantime"] }
